### ISSUE TITLE
Does not work with blueprints

### ISSUE THREAD

**Issue by yuri-karadzhov** [lykmapipo/sails-hook-validation#35]
Standard errors works with blueprints and return json error object, but sails-hook-validation return 500 error with no validation details by default.

**Comment by lykmapipo** [comment]
@yuri-karadzhov 

What version of  `sails-hook-validation` are you using?. 
If its latest version, may you please cross-check if you have setup model or local files correct.

**Comment by yuri-karadzhov** [comment]
Latest one and yes, i do set up models with right verification. I actually saw the right info in the console but on the client side there is only an 500 error.

**Comment by einfallstoll** [comment]
It actually worked sometime for me… I'm currently investigating the issue but expect it to be related to `sails` itself and not `sails-hook-validation` since the corresponding `Errors` object will be set on the error.

**Comment by einfallstoll** [comment]
I found the problem and was able to create a fix ( #40 ). There's a function in `waterline` that makes sure it's a `WLError` (waterline error) and if its not a `WLError` it'll "errorify" it (yep, that's the name of that function). Since the hook applies a patch to the `toJSON` function it will get overwritten at that point.

To make sure the `errorify` function just returns our patched error, I'll have to make sure the error is a valid `WLError` with a patched function.

TL;DR; my PR does the trick.

**Comment by lykmapipo** [comment]
@einfallstoll 

Thanks for the fix.

**Comment by lykmapipo** [comment]
v0.4.5 is out with the fix
